Makefile

```diff
@@ -0,0 +1,45 @@
+BUILD_CHANNEL?=local
+OS=$(shell uname)
+VERSION=v1.12.0
+GIT_REVISION = $(shell git rev-parse HEAD | tr -d '\n')
+TAG_VERSION?=$(shell git tag --points-at | sort -Vr | head -n1)
+GO_BUILD_LDFLAGS = -ldflags "-X 'main.Version=${TAG_VERSION}' -X 'main.GitRevision=${GIT_REVISION}'"
+GOPATH = $(HOME)/go/bin
+export PATH := ${GOPATH}:$(PATH) 
+SHELL := /usr/bin/env bash 
+
+default: setup build
+
+setup: install-dependencies
+
+install-dependencies:
+ifneq (, $(shell which brew))
+	brew update
```

Comment:
Brew is not needed to build this module, why are we running brew update?

Reply:
The Makefile I included is just a framework I was using in eng when creating modules. It helped making things consistent as I jumped from module to module, however, you are right that it is not necessary in this case. Would you like me to remove it from this PR?